lib/phoenix_storybook/components/icon.ex

```diff
@@ -83,6 +83,29 @@ defmodule PhoenixStorybook.Components.Icon do
     end
   end
 
+  @doc """
+  Local icons for internal phoenix_storybook usage.
+
+  ## Examples
+
+      <.local_icon name="book" class="text-blue-400"/>
+      <.local_icon name="book" style={:duotone} plan={:pro}/>
```

Comment:
This example is too _fontawesomish_
```suggestion
      <.local_icon name="hero-book" style={:mini}/>
```